src/com/seafile/seadroid2/ui/activity/BrowserActivity.java

```diff
@@ -1376,9 +1376,9 @@ public void showFile(File file) {
             return;
         }
 
-        String mime = MimeTypeMap.getSingleton().getMimeTypeFromExtension(suffix);
```

Comment:
the returned value of  `MimeTypeMap.getSingleton().getMimeTypeFromExtension` could be null sometimes